gulpfile.cjs

```diff
@@ -297,13 +256,13 @@ gulp.task('watch', function watch () {
 
 // The default task (called when you run `gulp`)
 gulp.task('default', gulp.series(
-  'clean',
+  clean,
   updateVersionFile,
   compileCommonJs,
   generateEntryFiles,
   compileEntryFiles,
   compileESModules, // Must be after generateEntryFiles
   writeCompiledHeader,
-  'browser',
-  'docs'
+  bundle,
+  generateDocs
```

Comment:
Just thought it worth mentioning, I set these to the string names of the tasks that I created above, so Gulp will run those tasks. That way, if one of the tasks changes (Like with the browser compiling changing from `bundle, minify` to just `bundle`) you only need to change the individual task, and not the one here as well.

Reply:
Yes, true. Formerly the `'browser'` task did two things: bundle and minify. There is only one left now: bundle, so I thought like let's remove this indirection. I think we can maybe remove `gulp.task('browser', bundle)` because it's not used anywhere externally from the package.json scripts. The task 'docs' only does one thing (generateDocs) but is actually used in the package.json scripts.

Reply:
Yeah, I initially created those tasks because I found that I kept needing to clean out things and only rebuild certain things when I was testing, so it made it a bit easier.

Reply:
Ah! Makes sense. If you like it better to call the gulp tasks instead of the function it's totally fine with me, for me both is ok.

Reply:
I don't really mind. It's a minor thing anyway, so might as well leave it as is for now.